tests/test_client.py

```diff
@@ -1151,6 +1285,57 @@ def test_chained_exceptions_with_no_cause(self):
         assert exceptions[0]['stacktrace'][0]['method'] == \
             'test_chained_exceptions_with_no_cause'
 
+    def test_chained_exceptions_with_no_cause_using_capture_cm(self):
```

Comment:
are the names of this test and the one below it swapped?

Reply:
yeah, not sure how that happened! 🤦